app/views/response_mailer/response_email.html.haml

```diff
@@ -0,0 +1,14 @@
+%html{xmlns: 'http://www.w3.org/1999/xhtml'}
+  %head
+    %meta{'http-equiv' => 'Content-type', content: 'text/html; charset=UTF-8'}
+      %body
+        %p
+          Uw patient met het e-mailadres
+          = mail_to "no_respondent_email_available"
```

Comment:
Misschien is het handig om hier i.p.v een placeholder de datum en tijd van de `InvitationSent` te gebruiken, dan is er in ieder geval íets om aan te geven om welk verzoek het ging.